rom/laghos.cpp

```diff
@@ -931,14 +941,20 @@ int main(int argc, char *argv[])
 
     // 2D Rayleigh-Taylor penetration distance
     int pd1_vdof = -1, pd2_vdof = -1;
```

Comment:
A brief comment here would be helpful, explaining that these are position/velocity DOFs of a particular point hard-coded for problem 7.

Reply:
Comment is added.